service/collector.go

```diff
@@ -183,7 +183,15 @@ LOOP:
 			col.logger.Info("Received shutdown request")
 			break LOOP
 		case <-ctx.Done():
-			col.logger.Error("Context done, terminating process", zap.Error(ctx.Err()))
+			col.logger.Info("Context done, terminating process")
```

Comment:
I would include the error anyway in this message, or is that the one that prints the stacktrace?

Reply:
Yeah the `zap.Error(ctx.Err())` prints the stack trace. I think if you don't wrap it in `zap.Error` it won't print the stack but rather just the error string. I would be ok doing that I think.